runners/direct-java/pom.xml

```diff
@@ -247,6 +247,14 @@
   </build>
 
   <dependencies>
+    <!-- jsr305 dependency must come before java-sdk-all, as java-sdk-all depends on
```

Comment:
Ok, the latest version makes the world better by:

1. Explicitly exclude `jsr305` from transitive dependencies
2. Switch all usages of `findbugs:jsr305` to `findbugs:annotations`

This works for our codebase. If other projects depend on Beam + JSR305 (and validate dependencies), they'll likely have the same issue. The workaround for them would be to depend on findbugs:annotations instead.